h2o-bindings/bin/custom/python/gen_gbm.py

```diff
@@ -3,6 +3,19 @@ def update_param(name, param):
         param['values'].remove('ordinal')
         return param
     return None  # param untouched
+ 
+def class_extensions():
+    """
+      Returns actual number of trees in GBM model (whether CV is enabled or not).
+
+      Type: ``float``
+      """
+    def ntree_actual(self):
```

Comment:
I dont like the name. Everywhere we use ntrees, my suggestions:
- ntrees_actual
- ntrees_built

Reply:
Taken from the ticket description. Will change :)

Reply:
done